src/connectionController.ts

```diff
@@ -418,7 +421,20 @@ export default class ConnectionController {
     }
 
     log.info('Successfully connected', { connectionId });
-    void vscode.window.showInformationMessage('MongoDB connection successful.');
+
+    if (prevConnectionId) {
+      const prevConnectionName = prevConnectionId
+        ? this.getSavedConnectionName(prevConnectionId)
+        : 'MongoDB server';
+
+      void vscode.window.showInformationMessage(
+        `Disconnected from ${prevConnectionName} and connected to ${nextConnectionName}.`
```

Comment:
Rather than the word `Changed`, how about `Set`?

The word `Changed` implies the presence of a connection before, but if you'd like a single message for both an initial connection and changing connection, I think `Set` is more fitting.